### PR TITLE
remove the 'x' button and replace with a dropdown button

### PR DESCRIPTION
**Pull request by bendesu**
I replaced the 'x' button with a dropdown button.
Now, when users click the dropdown button, it will show a list of parameters of jovian.commit().
Once users fill in all the parameters(it could be default value), and click the summit button, it will pass these parameters to Python code, and then summit to Jovian.

### CODE REVIEW

**Comment by aakashns** [comment]
Please add a screenshot of the UI here. 

Also, note the following:
* Wherever the user needs to provide one of multiple options, please provide a dropdown. E.g. env_type can be 'conda' or 'pip'.
* For boolean parameters like `secret`, please provide a checkbox.

**Comment by bendesu** [comment]
I have changed the view as the requirement.

Also, now the extension can stored all the parameters once the user use the dropdown option, and show all stored parameters when the user click it again.

**Comment by bendesu** [comment]
Screenshot

File: jovian/jovian_nb_ext/main.js [hunk 1]
```diff
@@ -344,20 +687,20 @@ define([
     );
 
     //toolbar button to remove the jovian extension
-    const remove_ext_action = {
-      icon: "fa-times",
+    const set_paras_ext_action = {
+      icon: "fa-angle-double-down",
       help: "Disable Jovian Extension",
```
Comment (PrajwalPrashanth) [inline jovian/jovian_nb_ext/main.js]: `help` key is the tooltip for the toolbar button. You can the add the appropriate tooltip here.

File: jovian/jovian_nb_ext/main.js [hunk 2]
```diff
@@ -344,20 +687,20 @@ define([
     );
 
     //toolbar button to remove the jovian extension
-    const remove_ext_action = {
-      icon: "fa-times",
+    const set_paras_ext_action = {
```
Comment (PrajwalPrashanth) [inline jovian/jovian_nb_ext/main.js]: Can you use `params` instead of `paras` in all instances

File: jovian/jovian_nb_ext/main.js (hunk 1 above)
Comment (bendesu) [reply jovian/jovian_nb_ext/main.js]: Yeah! sure, I will change by tomorrow. Thanks for sharing me the way of testing. I will also do that.

File: jovian/jovian_nb_ext/main.js (hunk 2 above)
Comment (bendesu) [reply jovian/jovian_nb_ext/main.js]: Sure, I will change this, and also I will consider another to store parameters if there's any other better way.

**Pull request merged**